src/main/webapp/script.js

```diff
@@ -326,9 +323,86 @@ function createAuthenticationButton(authenticationUrl, buttonStyle, buttonText,
  * Create message for the approval page that shows approval status directed to mentor or approver.
  * TODO(oumontiel): Create content for message.
  */
-function createApprovalMessage(example) {
+function createApprovalMessage(reviewStatus, spanColor, spanMessage) {
+  // Create elements for messages that will be added and the span that will be appended.
+  const approvalSubtitleElement = document.getElementById('approval-subtitle');
+  const approvalSmallTextElement = document.getElementById('approval-small-text');
+
+  // Create span element that will be appended to subtitle.
+  const approvalSpanElement = document.createElement('span');
+  approvalSpanElement.setAttribute('class', spanColor);
+  approvalSpanElement.textContent = spanMessage;
+  
+  switch(reviewStatus) {
+    case 1:
+      approvalSubtitleElement.appendChild(approvalSpanElement);
+      approvalSubtitleElement.appendChild(document.createTextNode(
+          ' Your review has been approved!'));
+      approvalSmallTextElement.appendChild(document.createTextNode(
+          'Your username will now show a verified icon to show off your experience.'));
+      break;
+    case 2:
+      approvalSubtitleElement.appendChild(document.createTextNode(
+        'We\'re really sorry, your review has been '));
+      approvalSubtitleElement.appendChild(approvalSpanElement);
+      approvalSubtitleElement.appendChild(document.createTextNode('.'));
+      approvalSmallTextElement.appendChild(document.createTextNode(
+          'If you think this is a mistake, click the button to update your information.'));
+
+      // Create and append button to redirect to verification page.
+      const redirectButton = document.createElement('button');
+      redirectButton.type = 'button';
+      redirectButton.setAttribute('class', 'btn btn-success');
+      redirectButton.onclick = function() {
+        window.location.replace('verification.html');
+      };
+      redirectButton.innerHTML = 'Update information';
+      const approvalMessageElement = document.getElementById('rejected-button');
+      approvalMessageElement.appendChild(redirectButton);
+
```

Comment:
Perhaps erase this blank to keep consistency with rest of cases.

Reply:
I think this was caused because you merged this with another branch, but I undid that merge so it's now good. Also, this function underwent a big change because of Shaar's request, so it looks different now.